docs/mkdocs.yml

```diff
@@ -0,0 +1,20 @@
+site_name: Longleaf Documentation
```

Comment:
mkdocs.yml should be at the root of the repo

Reply:
Done.